config/foreman-proxy-content.migrations/190710195757-force_no_prefix_false.rb

```diff
@@ -0,0 +1 @@
+scenario.delete(:no_prefix)
```

Comment:
This is different than the change I did downstream, because Travis (rightfully) yelled at me :)